pom.xml

```diff
@@ -24,15 +24,16 @@
         <module>pojo-event-annotation-plugin</module>
         <module>standard-test-catalog</module>
         <module>example-pojo-schema</module>
+        <module>example-schema-to-pojo-gen-test</module>
     </modules>
 
     <properties>
         <common-bom.version>1.27.0</common-bom.version>
-        <framework-api-version>2.1.2</framework-api-version>
+        <framework-api-version>3.0.0-M14-SNAPSHOT</framework-api-version>
```

Comment:
the framework-api version needs to be a current release version...not SNAPSHOT or milestone release